vault-core/src/main/java/org/apache/jackrabbit/vault/fs/impl/io/DocViewImporter.java

```diff
@@ -1200,28 +1200,37 @@ private boolean isPropertyProtected(@NotNull EffectiveNodeType effectiveNodeType
         return effectiveNodeType.getApplicablePropertyDefinition(npResolver.getJCRName(docViewProperty.getName()), docViewProperty.isMultiValue(), docViewProperty.getType()).map(PropertyDefinition::isProtected).orElse(false);
     }
 
+    private String getLexicalForm(Name name) throws RepositoryException {
```

Comment:
OTOH, we could just always use "getJCRName()"...

Reply:
IMHO this additional lookup costs some performance.